saleor/discount/tasks.py

```diff
@@ -10,48 +10,63 @@
 from ..celeryconf import app
 from ..graphql.discount.mutations.utils import CATALOGUE_FIELD_TO_TYPE_NAME
 from ..plugins.manager import get_plugins_manager
+from ..product.tasks import update_products_discounted_prices_of_catalogues_task
 from .models import Sale
 from .utils import CATALOGUE_FIELDS, CatalogueInfo
 
 task_logger = get_task_logger(__name__)
 
 
 @app.task
-def send_sale_toggle_notifications():
-    """Send the notification about starting or ending sales."""
+def handle_sale_toggle():
```

Comment:
Should we leave the old task for situations where one of the tasks is still pending in the queue, and we have already upgraded the celery to the latest version without this task?

Reply:
Good point! Thanks @fowczarek

Reply:
Done